### ISSUE TITLE
Alt text missing from people directory images pulled in with content syndicate

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/ip.wsu.edu#80]
https://github.com/washingtonstateuniversity/ip.wsu.edu/blob/master/functions.php#L252-L255

This should at least have `alt=""`. It may be appropriate to have a more complete description.
